vscode-extension/src/util.ts

```diff
@@ -295,3 +295,19 @@ export function urlJoin(...args) {
   return normalize(parts);
 }
 //#endregion
+
+
+/**
+ * AIConfig Vscode extension has a dependency on the Python extension. 
+ * This function retrieves and returns the path to the current python interpreter.
+ * @returns the path to the current python interpreter
+ */
+export async function getPythonPath(): Promise<string> {
+  const pythonExtension = vscode.extensions.getExtension("ms-python.python");
+  if (!pythonExtension.isActive) {
+    await pythonExtension.activate();
+  }
+  
+  const pythonPath = pythonExtension.exports.settings.getExecutionDetails().execCommand[0];
```

Comment:
How did you determine this?

Reply:
err, @saqadri gave me `pythonExtension.exports.settings.getExecutionDetails().execCommand`. This returns a list. When tested locally, this list contained one element, the interpreter path.